.github/workflows/build_release_action.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build and release ruby gem
+
+on:
+  pull_request: 
+    branches: [ master ]
+  push:
+  workflow_dispatch:
+
+jobs:
+  build-release-pipeline:
+    runs-on: ubuntu-latest
+    container: ruby:2.6.3
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup
+        run: |
+          gem install bundler:1.17.3
+          bundle install
+
+      - name: Build
+        id: build
+        if: success() && github.ref == 'refs/heads/master'
+        run: |
+          bundle exec rake build
+          echo "::set-output name=gem_version::$(cat ./lib/request_local_cache/version.rb | grep VERSION | sed "s/'//g" | sed 's/"//g' | awk '{ print "v"$3 }')"
```

Comment:
We can add a rake task that does `puts RequestLocalCache::VERSION` to avoid doing this.